FU_NSSDB/NSSdbOps.cs

```diff
@@ -211,16 +211,19 @@ private string getSQL(SQLType type)
                     /*results = @"SELECT DISTINCT sl.StatLabel 
                                 FROM ([Parameters] p 
                                 LEFT JOIN StatLabel sl ON ( p.StatisticLabelID = sl.StatisticLabelID))";*/
-                    results = @"SELECT DISTINCT (0-1) as ID, sl.StatLabel as Code, sl.Definition as Description, sl.StatisticLabel as Name
-                                FROM (DepVars s
+                    results = @"SELECT DISTINCT (0-1) as ID, sl.StatLabel as Code, sl.Definition as Description, sl.StatisticLabel as Name, ut.MetricAbbrev as MetricAbbrev, ut.EnglishAbbrev as EnglishAbbrev, st.StatisticTypeCode as StatType
+                                FROM ((DepVars s
                                 LEFT JOIN StatLabel sl on (s.StatisticLabelID = sl.StatisticLabelID))
+                                LEFT JOIN Units ut on (sl.UnitID = ut.UnitID))
                                 LEFT JOIN StatType st on (sl.statisticTypeID = st.StatisticTypeID)
                                 WHERE st.DefType = 'BC';";
                     break;
                 case SQLType.e_statisticgrouptype:
                     //results = @"SELECT DISTINCT st.StatisticTypeCode FROM StatType st WHERE st.DefType ='FS'";
-                    results = @"SELECT DISTINCT (0-1) as ID, st.StatisticTypeCode as Code, st.StatisticType as Name 
-                                FROM StatType st WHERE st.DefType ='FS'";
+                    //results = @"SELECT DISTINCT (0-1) as ID, st.StatisticTypeCode as Code, st.StatisticType as Name 
```

Comment:
Is there a reason we are keeping this?

Reply:
I think I was nervous about fully removing it, but you're right - this can be removed! 😆